acra-translator/go-translator-demo/README.md

```diff
@@ -0,0 +1,24 @@
+# cossacklabs go-translator-demo
+
+### Pre requirements
+
+    go version go1.18.0 or higher 
+    docker version 19.03.8
+    swagg # for swagger generatetion
+
+### Swagger
+
+Swagger is automatically generated from comments in source code using `swag`. To install `swag` locally:
+
+```
+go get -u github.com/swaggo/swag/cmd/swag
+
+# 1.18 or newer
+go install github.com/swaggo/swag/cmd/swag@latest
```

Comment:
let's freeze the swag version that was used in tests. Now software updates so frequently that we will have down examples in 3 weeks )